appveyor.yml

```diff
@@ -1,4 +1,4 @@
-version: 1.0.{build}.0
+version: 1.1.{build}.0
```

Comment:
We are increasing version numbers when we merge into the master, right before the release. Can you please revert the version change?